### PR TITLE
Edits to module_dialogs.py and module_strings.py

### PR DESCRIPTION
**Pull request by airtroop392**
See individual commits for documentation on changes.

- 68994a41115b4f7a5fd1ac2f5d897cff6e42f1f0 modifies module_dialogs.py only.
- f4bee00a36c13d7e48a4b38c844ecde45389b99c modifies module_strings.py only.
- 97f71b48a34a0c170a3c520dccbdb4b629d6cb23 is an accidental commit and should not change module_strings.py in any way (currently up to date). Still getting used to GitHub Desktop- sorry!

### CODE REVIEW

File: module_strings.py [hunk 1]
```diff
@@ -2209,77 +2209,77 @@
...
-  ("spaceship_tie_fighter_desc", "The Tie Fighter is a relatively modern fighter that makes use of ^the last propulsion technology."),
+  ("spaceship_imperial_trade_frigate_desc", "The standard Imperial Trade Frigate combines acceptable engine speed ^and lots of space for either troops or cargo."),
+  ("spaceship_tie_fighter_name", "Imperial TIE Fighter"),
+  ("spaceship_tie_fighter_desc", "The TIE Fighter is a bare-bones fighter that is only noticeable^ for having the latest in propulsion technology."),
   ("spaceship_imperial_shuttle_name", "Lambda-class T-4a Shuttle"),
   #("spaceship_imperial_shuttle_desc", "A true beauty, the Imperial Shuttle combines speed with troop support capabilites. ^^It has one troop compartment with room for two more and a MK-I Medical bay ^which can be upgraded to MK-II. ^^The current MK-I scanners could be upgraded to MK-III if you require a larger scanning range."),
-  ("spaceship_imperial_shuttle_desc", "A true beauty, the Lambda Shuttle combines speed ^with troop support capabilites."),
+  ("spaceship_imperial_shuttle_desc", "A true beauty, the Lambdap-class Shuttle combines engine speed ^with troop support capabilites and capacity."),
```
Comment (Swyter) [inline module_strings.py:2226]: @airtroop392 I think that _lambdap_  is a typo, I can correct it later. Everything until here looks pretty good.

File: module_strings.py [hunk 2]
```diff
@@ -2209,77 +2209,77 @@
...
   ("spaceship_hutt_trade_name", "Hutt Trade Ship"),
   ("spaceship_hutt_trade_desc", "Though quite slow, the standard Hutt Trade Ship makes use ^of an excellent trade computer and cargo hold."),
   ("spaceship_hutt_patrol_name", "Hutt Patrol Ship"),
   ("spaceship_hutt_patrol_desc", "The standard Hutt Patrol Ship is excellent for tracking transports ^on short distances."),
   #other
   ("spaceship_z95_name", "Z-95 HeadHunter"),
   #("spaceship_z95_desc", "The z-95 Headhunter is a fast and agile craft which will see you safely to ^all corners of the galaxy. ^^Its powerful MK-III engines can be upgraded to MK-V to make it the fastest ^ship out there, unless you believe that nonsense about the Millenium Falcon.^^It has MK-I scanners with the option to upgrade to MK-III."),
-  ("spaceship_z95_desc", "The z-95 Headhunter is a fast and agile craft which will take you safely to ^all corners of the galaxy."),
+  ("spaceship_z95_desc", "The z-95 Headhunter is a fast and agile craft that can run from any fight, ^although it doesn't have the resources to win one."),
```
Comment (Swyter) [inline module_strings.py:2257]: @airtroop392 I like this one a lot. :)

File: module_strings.py (hunk 1 above)
Comment (airtroop392) [reply module_strings.py:2226]: Yes, it is. Thanks! :)

**Comment by Swyter** [comment]
@airtroop392 Okay, looking great. Let's merge it!

**Comment by Swyter** [comment]
@airtroop392 I have just committed the changes. With the fixes outlined in:
https://github.com/swconquest/swconquest-fix-dialogs/commits/master

Be sure to pull and rebase your changes before opening the pull request. That should be it.

Should appear in the Steam Workshop in a few minutes. Sit tight.

**Pull request merged**